dask/utils.py

```diff
@@ -498,6 +498,25 @@ def wrapper(method):
             if doc is None:
                 doc = ''
 
+            # Insert disclaimer near the top, after the title.
+            if doc:
+                copied = (
+                    "This docstring was copied from %s.%s.%s\n"
+                    "Some inconsistencies with the Dask version may exist. "
+                    % (original_klass.__module__, original_klass.__name__,
+                        method_name)
+                )
+
+                lines = doc.split('\n')
+                empty = 0
+                for i, line in enumerate(lines):
+                    if not line.strip():
+                        empty += 1
+                    if empty == 2:
+                        break
```

Comment:
Why not just use:

```python
i = doc.find('\n\n')
if i == -1:
    # No blank lines found, add our warning to the end
    doc = '\n\n'.join([doc, copied])
else:
    # Insert our warning
    doc = ''.join([doc[:i + 2], copied, doc[i+2:]])
```

Avoids iterating over the cost of splitting the whole docstring and iterating over the lines.

Reply:
A more full version that also handles indentation of our warning:

```python
import re


def insert_warning(doc):

    l1 = "This docstring was copied from foo.bar.baz\n"
    l2 = "Some inconsistencies with the Dask version may exist."

    i = doc.find('\n\n')
    if i == -1:
        # No blank lines found
        # Add our warning to the end with no blank line after
        doc = '\n\n'.join([doc, '\n\n', l1, l2])
    else:
        # Insert our warning
        head = doc[:i + 2]
        tail = doc[i + 2:]
        # Indentation of next line
        indent = re.match(r'\s*', tail).group(0)
        # Insert the warning, indented, with a blank line before and after
        doc = ''.join([
            head,
            indent, l1,
            indent, l2, '\n\n',
            tail
        ])
    return doc
```

Reply:
Stolen.  Thanks!